Fonts/PTSenif-Bold/stylesheet.css

```diff
@@ -0,0 +1,8 @@
+/*! Generated by Font Squirrel (https://www.fontsquirrel.com) on September 26, 2020 */
+
+@font-face {
+    font-family: 'pt_serifbold';
+    src: url('ptserif-bold-webfont.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6241956)